classes/locations.php

```diff
@@ -623,9 +628,14 @@ function dragDropUpdateAddress() {
 		function save_post_meta( $post ) {
 			// This function saves the geo data as well as the additional info.
 
-			global $simple_map, $current_screen;
+        if ( ! isset( $_POST['sm_save_postmeta_nonce_field'] ) || ! wp_verify_nonce( $_POST['sm_save_postmeta_nonce_field'], 'sm_save_postmeta_nonce_action' )){
+    return;
+		}
+
+		global $simple_map, $current_screen;
 
 			// Bail if we're not editing a location.
+			// This should be replaced by a nonce!!!
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->